packages/app-staking/src/md/basic.md

```diff
@@ -0,0 +1,32 @@
+# Staking
+
+Welcome to the staking module. Here you can see an overview of all the validators on the network and participate in the network as either as validtor (running a node that helps secure the network), or a nominator (adding funds to help secure the network).
+
+Stakers are nominators earn rewards: at the end of an era the rewards are split between the validators and nominators for that validator, in proportion to the amount bonded by each individual. Simmilarly, at the end of a session set of new validators are selected.
```

Comment:
Did you mean: "Stakers **and** nominators"? I'd keep the notion of "validator" instead of "staker" in that case.
```suggestion
Validators and nominators earn rewards at the end of an era: the rewards are split between the validators and nominators for that validator, in proportion to the amount bonded by each individual.
```
I think " Similarly, at the end of a session set of new validators are selected." doesn't belong to this section dedicated to rewards.